src/artikler/sok-sosialhjelp/komponenter/kommunesok/Kommunesok.tsx

```diff
@@ -71,34 +87,46 @@ const KommuneSok: React.FC<Props> = ({
                 <b>{ledetekst}</b>
             </Normaltekst>
             <br />
-            <NavAutocomplete
-                placeholder={placeholderTekst}
-                suggestions={suggestions}
-                ariaLabel={ariaLabel}
-                id="kommunesok"
-                onSelect={(suggestion: Suggestion) => onSelect(suggestion)}
-                onReset={() => onReset()}
-            />
-            {currentSuggestion && soknadTilgjengelig && (
+            {kommuneNrService.restStatus === REST_STATUS.OK &&
+            tilgjengeligeKommunerService.restStatus === REST_STATUS.OK ? (
+                <NavAutocomplete
+                    placeholder={placeholderTekst}
+                    suggestions={suggestions}
+                    ariaLabel={ariaLabel}
+                    id="kommunesok"
+                    onSelect={(suggestion: Suggestion) => onSelect(suggestion)}
+                    onReset={() => onReset()}
+                />
+            ) : (
+                <NavFrontendSpinner />
+            )}
+            {currentSuggestion && kommuneHarSoknad && kommuneHarInnsyn && (
+                <div style={{textAlign: "left", paddingTop: "1rem"}}>
+                    <div className="kommunesok_tilbakemelding ">
+                        <Normaltekst>
+                            {soknadOgInnsynTilgjengeligTekst}
+                        </Normaltekst>
+                    </div>
+                </div>
+            )}
+            {currentSuggestion && kommuneHarSoknad && !kommuneHarInnsyn && (
                 <div style={{textAlign: "left", paddingTop: "1rem"}}>
                     <div className="kommunesok_tilbakemelding ">
                         <Normaltekst>
-                            {soknadTilgjengeligTekst} {currentSuggestion.value}
+                            {soknadTilgjengeligUtenInnsynTekst}
                         </Normaltekst>
                     </div>
                 </div>
             )}
-            {!soknadTilgjengelig && currentSuggestion && (
+            {!kommuneHarSoknad && currentSuggestion && (
                 <div style={{textAlign: "left", paddingTop: "1rem"}}>
                     <div
                         className="kommunesok_tilbakemelding "
                         style={{paddingTop: "0.5rem", marginBottom: "0"}}
```

Comment:
Er det en grunn til at denne er stylet litt annerledes enn de over? 

Hvis ikke, kunne man ha hatt if-ene inni <NormalTekst>? For å slippe så mye duplikat-kode av boksen rundt (om den er lik) 
(Vet det ikke er du som har laget dette, og det trengs ikke gjøres nå. Er mest nysgjerrig :) )

Reply:
Godt spørsmål, som jeg ikke vet svaret på. Foreslår at vi tar det i en egen PR, og samtidig trekker ut styling til egen less, slik at vi unngår styling både inline og i less-filer.